platform/uwp/os_uwp.h

```diff
@@ -58,13 +58,13 @@ class OS_UWP : public OS {
 			CHAR_EVENT_MESSAGE
 		};
 
-		bool alt, shift, control;
-		MessageType type;
-		bool pressed;
-		unsigned int scancode;
-		unsigned int physical_scancode;
-		unsigned int unicode;
-		bool echo;
+		bool alt = false, shift = false, control = false;
+		MessageType type = KEY_EVENT_MESSAGE;
+		bool pressed = false;
+		unsigned int scancode = 0;
+		unsigned int physical_scancode = 0;
+		unsigned int unicode = 0;
+		bool echo = false;
```

Comment:
See `OS_UWP()` constructor and make sure that the values here are in sync, and remove them from the constructor. (The constructor can't be removed in this case but all primitive value assignments can be.)